commands/monitoring/pipeline/delete.go

```diff
@@ -0,0 +1,88 @@
+package pipeline
+
+import (
+	"context"
+	"fmt"
+	"github.com/ionos-cloud/ionosctl/v6/commands/monitoring/pipeline/completer"
+	"github.com/ionos-cloud/ionosctl/v6/internal/client"
+	"github.com/ionos-cloud/ionosctl/v6/internal/constants"
+	"github.com/ionos-cloud/ionosctl/v6/internal/core"
+	"github.com/ionos-cloud/ionosctl/v6/internal/printer/jsontabwriter"
+	"github.com/ionos-cloud/ionosctl/v6/pkg/confirm"
+	"github.com/ionos-cloud/ionosctl/v6/pkg/functional"
+	"github.com/ionos-cloud/sdk-go-bundle/products/monitoring/v2"
+	"github.com/spf13/viper"
+)
+
+func MonitoringDeleteCmd() *core.Command {
+	cmd := core.NewCommand(context.Background(), nil, core.CommandBuilder{
+		Namespace: "monitoring",
+		Resource:  "pipeline",
+		Verb:      "delete",
+		Aliases:   []string{"del", "d"},
+		ShortDesc: "Delete a pipeline",
+		Example:   "ionosctl monitoring pipeline delete --pipeline-id ID --location de/txl",
+		PreCmdRun: func(c *core.PreCommandConfig) error {
+			return core.CheckRequiredFlagsSets(c.Command, c.NS, []string{constants.ArgAll}, []string{constants.FlagPipelineID})
+		},
+		CmdRun: func(c *core.CommandConfig) error {
+			if all := viper.GetBool(core.GetFlagName(c.NS, constants.ArgAll)); all {
+				return deleteAll(c)
+			}
+
+			monitoringId := viper.GetString(core.GetFlagName(c.NS, constants.FlagPipelineID))
+			z, err, _ := client.Must().Monitoring.PipelinesApi.PipelinesFindById(context.Background(), monitoringId).Execute()
+			if err != nil {
+				return fmt.Errorf("failed getting pipeline by id %s: %w", monitoringId, err)
+			}
+			yes := confirm.FAsk(c.Command.Command.InOrStdin(),
+				fmt.Sprintf("Are you sure you want to delete pipeline with name: %s, id: %s ", z.Properties.Name, z.Id),
+				viper.GetBool(constants.ArgForce))
+			if !yes {
+				return fmt.Errorf(confirm.UserDenied)
+			}
+
+			_, newerr := client.Must().Monitoring.PipelinesApi.PipelinesDelete(context.Background(), monitoringId).Execute()
+			if newerr != nil {
+				return fmt.Errorf("failed deleteing the pipeline: \n%w", newerr)
```

Comment:
```suggestion
				return fmt.Errorf("failed deleting the pipeline: %w", newerr)
```

stop adding newlines to errors